library/kani/src/lib.rs

```diff
@@ -21,6 +21,10 @@ pub mod vec;
 pub use arbitrary::Arbitrary;
 #[cfg(feature = "concrete_playback")]
 pub use concrete_playback::concrete_playback_run;
+#[cfg(not(feature = "concrete_playback"))]
+/// NOP `concrete_playback` for type checking during verification mode.
+pub fn concrete_playback_run<F: Fn()>(_: Vec<Vec<u8>>, _: F) {}
```

Comment:
Can you please add an `unreachable!("Concrete playback does not work during verification")` or something like that, just in case users ever reach this code.

Reply:
Done [c057854](https://github.com/model-checking/kani/pull/2476/commits/c057854d26cdf961d3ed778cdf9e1582e0318dd8)